fixtures/cameo/ts-200-fc.json

```
@@ -0,0 +1,627 @@
+          "colors": [
+            "#ffff00"
+          ]
+        },
+        {
+          "dmxRange": [38, 45],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": [
+            "#00ff00"
+          ]
+        },
+        {
+          "dmxRange": [46, 53],
+          "type": "ColorPreset",
+          "comment": "Turquoise",
+          "colors": [
+            "#40e0d0"
+          ]
+        },
+        {
+          "dmxRange": [54, 61],
+          "type": "ColorPreset",
+          "comment": "Cyan",
+          "colors": [
+            "#00ffff"
+          ]
+        },
+        {
+          "dmxRange": [62, 69],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": [
+            "#0000ff"
+          ]
+        },
+        {
+          "dmxRange": [70, 77],
+          "type": "ColorPreset",
+          "comment": "Lavender",
+          "colors": [
+            "#e6e6fa"
+          ]
+        },
+        {
+          "dmxRange": [78, 85],
+          "type": "ColorPreset",
+          "comment": "Mauve",
+          "colors": [
+            "#e0b0ff"
+          ]
+        },
+        {
+          "dmxRange": [86, 93],
+          "type": "ColorPreset",
+          "comment": "Magenta",
+          "colors": [
+            "#ff00ff"
+          ]
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "Pink",
+          "colors": [
+            "#ffc0cb"
+          ]
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "Warm White 3200K",
+          "colors": [
+            "#fff1e0"
+          ]
+        },
+        {
+          "dmxRange": [110, 119],
+          "type": "ColorPreset",
+          "comment": "White 5600K",
+          "colors": [
+            "#fffffb"
+          ]
+        },
+        {
+          "dmxRange": [120, 125],
+          "type": "ColorPreset",
+          "comment": "Cold White 8500K",
+          "colors": [
+            "#f4fdff"
+          ]
+        },
+        {
+          "dmxRange": [126, 127],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speed": "stop",
+          "comment": "Color Jumping stop"
+        },
+        {
+          "dmxRange": [128, 191],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "parameterStart": 1,
+          "parameterEnd": 12,
+          "comment": "Color Jumping speed slow -> fast / Color 1 -> 12"
+        },
+        {
+          "dmxRange": [192, 255],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "parameterStart": 1,
+          "parameterEnd": 12,
+          "comment": "Color Fading speed slow -> fast / Color 1 -> 12"
+        }
+      ]
+    },
+    "Color Mode": {
```

```suggestion
    "Color Mode Crossfade": {
```

I was confused by these capabilities at first, but it turns out the channel name was just missing its final word from the manual.